src/services/ServiceService.js

```diff
@@ -0,0 +1,15 @@
+export default class ServiceService {
+    static instance = null;
```

Comment:
try importing the API root here and using it for the fetch
`import { API_ROOT } from './api-config';`